resources/META-INF/plugin.xml

```diff
@@ -694,6 +697,12 @@
                     serviceImplementation="io.flutter.view.FlutterView"
                     overrides="false"/>
 
+    <toolWindow id="Flutter PerfTools" anchor="right" icon="FlutterIcons.Flutter_13"
```

Comment:
Is this dead code? 
one thing that would be nice for his review is to send a few screenshots of your new UI in action. that way I know up front whether you are expecting a new tool window or if you are using the existing inspector perf tool window.